### PR TITLE
Update dataset.py_change the dataloader

### PR DESCRIPTION
**Pull request by liviuslw**
The order of the dataloader's label and image does not correspond to the order in train.py.

### CODE REVIEW

**Comment by weiaicunzai** [comment]
I switched to  the dataset module of pytorch instead my own dataset.py, so no need to change this line, you could do it locally, and use my dataset.py to train the network, it will also work. thanks for your contribution.
